tests/cases/expr/clamp.slint

```diff
@@ -0,0 +1,41 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+export component TestCase {
+    out property <float> t1: clamp(42.0, 10.0, 53.0);
+    out property <float> t2: clamp(42.0, 43.0, 53.0);
+    out property <float> t3: clamp(42.0, 10.0, 41.0);
+
+    r := Rectangle {
+        property <int> max: 42;
+        property <int> xx: Math.clamp(5, 2, 3) + max;
+    }
+    out property <bool> test: r.xx == 42 + 3;
```

Comment:
could you also test the value of t1, t2, t3 in there, so we test the interpreter?